src/core/lib/security/credentials/s2a/s2a_credentials.h

```diff
@@ -0,0 +1,68 @@
+/*
+ *
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef GRPC_CORE_LIB_SECURITY_CREDENTIALS_S2A_S2A_CREDENTIALS_H
+#define GRPC_CORE_LIB_SECURITY_CREDENTIALS_S2A_S2A_CREDENTIALS_H
+
+#include <grpc/grpc_security.h>
+#include <grpc/support/port_platform.h>
+#include "src/core/lib/security/credentials/credentials.h"
+#include "src/core/lib/security/credentials/s2a/grpc_s2a_credentials_options.h"
+
+namespace experimental {
+
+constexpr char kGrpcS2ACredentialsType[] = "s2a";
+
+/** The main client-side struct for securing a gRPC connection using S2A. **/
+class grpc_s2a_credentials final : public grpc_channel_credentials {
```

Comment:
Same comment RE adding "client" in the name.

Reply:
My same comment applies here: see [`alts_credentials.h`](https://github.com/grpc/grpc/blob/master/src/core/lib/security/credentials/alts/alts_credentials.h) or `spiffe_credentials.h`](https://github.com/grpc/grpc/blob/master/src/core/lib/security/credentials/tls/spiffe_credentials.h).